abletonosc/clip.py

```diff
@@ -106,9 +108,53 @@ def clip_callback(params: Tuple[Any]) -> Tuple:
             self.osc_server.add_handler("/live/clip/set/%s" % prop,
                                         create_clip_callback(self._set_property, prop))
 
+        def clip_get_notes_helper(clip, params: Tuple[Any] = ()):
+            if clip is None:
+                self.logger.info("Trying to get notes from an empty clip")
+                return ()
+            # Define default values
+            estimated_min_from_time = -16000
+            estimated_max_time_span = 1000000 # Ableton clip max length is 24 hours. This is more than enough at over 200bpm
+            # These numbers were came up after a bunch of try and error. They look arbitrary but it works. 
+            # I have tried different comnination of min and max time including using sys.maxsize, sys.float_info.min, sys.float_info.max
+            # clip.end_marker, clip_start_marker... but those don't work well. Notes are still missing
+            # https://github.com/ideoforms/AbletonOSC/issues/86
+
+
+            # Check if parameters are provided in the params tuple
+            if len(params) == 4:
```

Comment:
Should flag an error if `params` has a length other than 4, as otherwise this would silently fail

Reply:
@ideoforms In the `else` statement, default parameters were used. Would you prefer to throw an error instead?